src/main/webapp/script.js

```diff
@@ -226,6 +231,14 @@ function getOwnership(data) {
   }
 }
 
+/**
+ * @param {!Object} data
+ * @return {number} School ID.
+ */
+function getId(data) {
```

Comment:
Nit: Let's name this a bit more specifically, since it's hard to know what ID this might be. I think `getSchoolId` would be an accurate name.